### PR TITLE
docs: fix code comment err in Model Join case

### PR DESCRIPTION
**Pull request by BugKillerPro**
fix code comment err in Model Join case

### CODE REVIEW

**Comment by BugKillerPro** [comment]
@gqcn 
`
// LeftJoin does "LEFT JOIN ... ON ..." statement on the model.
// The parameter `table` can be joined table and its joined condition,
// and also with its alias name.
//
// Eg:
// Model("user").LeftJoin("user_detail", "user_detail.uid=user.uid")
// Model("user", "u").LeftJoin("user_detail", "ud", "ud.uid=u.uid")
// Model("user", "u").LeftJoin("SELECT xxx FROM xxx AS a", "a.uid=u.uid").
func (m *Model) LeftJoin(table ...string) *Model {
	return m.doJoin("LEFT", table...)
}
`
Eg:Model("user", "u").LeftJoin("SELECT xxx FROM xxx AS a", "a.uid=u.uid").
got Sql:  select * from user u left join (SELECT xxx FROM xxx AS a) on a.uid=u.uid; 
and with err : "Every derived table must have its own alias"

if change above Eg to:Model("user", "u").LeftJoin("SELECT xxx FROM xxx","a", "a.uid=u.uid").
we got right Sql: select * from user u left join (SELECT xxx FROM xxx ) AS a on a.uid=u.uid;

**Comment by gqcn** [comment]
> @gqcn
> 
> ```go
> // LeftJoin does "LEFT JOIN ... ON ..." statement on the model.
> // The parameter `table` can be joined table and its joined condition,
> // and also with its alias name.
> //
> // Eg:
> // Model("user").LeftJoin("user_detail", "user_detail.uid=user.uid")
> // Model("user", "u").LeftJoin("user_detail", "ud", "ud.uid=u.uid")
> // Model("user", "u").LeftJoin("SELECT xxx FROM xxx AS a", "a.uid=u.uid").
> func (m *Model) LeftJoin(table ...string) *Model {
> 	return m.doJoin("LEFT", table...)
> }
> ```
> 
> Eg:Model("user", "u").LeftJoin("SELECT xxx FROM xxx AS a", "a.uid=u.uid"). got Sql: ` select * from user u left join (SELECT xxx FROM xxx AS a) on a.uid=u.uid;` and with err : `"Every derived table must have its own alias"`
> 
> if change above Eg to:Model("user", "u").LeftJoin("SELECT xxx FROM xxx","a", "a.uid=u.uid"). we got right Sql: `select * from user u left join (SELECT xxx FROM xxx ) AS a on a.uid=u.uid;`

Awesome! Would you please post another pr to fix these comments?

**Comment by BugKillerPro** [comment]
Would you please update the parameter name as well?
Done

**Comment by BugKillerPro** [comment]
> > @gqcn
> > ```go
> > // LeftJoin does "LEFT JOIN ... ON ..." statement on the model.
> > // The parameter `table` can be joined table and its joined condition,
> > // and also with its alias name.
> > //
> > // Eg:
> > // Model("user").LeftJoin("user_detail", "user_detail.uid=user.uid")
> > // Model("user", "u").LeftJoin("user_detail", "ud", "ud.uid=u.uid")
> > // Model("user", "u").LeftJoin("SELECT xxx FROM xxx AS a", "a.uid=u.uid").
> > func (m *Model) LeftJoin(table ...string) *Model {
> > 	return m.doJoin("LEFT", table...)
> > }
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Eg:Model("user", "u").LeftJoin("SELECT xxx FROM xxx AS a", "a.uid=u.uid"). got Sql: ` select * from user u left join (SELECT xxx FROM xxx AS a) on a.uid=u.uid;` and with err : `"Every derived table must have its own alias"`
> > if change above Eg to:Model("user", "u").LeftJoin("SELECT xxx FROM xxx","a", "a.uid=u.uid"). we got right Sql: `select * from user u left join (SELECT xxx FROM xxx ) AS a on a.uid=u.uid;`
> 
> Awesome! Would you please update the parameter name as well?
@gqcn 
Done!

**Review by gqcn — CHANGES_REQUESTED** [review]

File: database/gdb/gdb_model_join.go [hunk 1]
```diff
@@ -30,7 +30,7 @@ func isSubQuery(s string) bool {
 // Eg:
 // Model("user").LeftJoin("user_detail", "user_detail.uid=user.uid")
 // Model("user", "u").LeftJoin("user_detail", "ud", "ud.uid=u.uid")
-// Model("user", "u").LeftJoin("SELECT xxx FROM xxx AS a", "a.uid=u.uid").
+// Model("user", "u").LeftJoin("SELECT xxx FROM xxx","tableAliasOrSubquery", "tableAliasOrSubquery.uid=u.uid").
 func (m *Model) LeftJoin(table ...string) *Model {
```
[the remark this answers is not in the document]
Comment (BugKillerPro) [reply database/gdb/gdb_model_join.go]: done

**Review by gqcn — APPROVED** [review]

**Pull request merged**